rfcs/0006-or-condition.md

```diff
@@ -0,0 +1,101 @@
+# RFC Proposal
+[RFC Proposal]: #rfc-proposal
+
+* RFC Proposal ID: `0006-or-condition`
+* Start Date: 2020-08-21
+* RFC PR: [https://github.com/CMSgov/beneficiary-fhir-data/pull/344](https://github.com/CMSgov/beneficiary-fhir-data/pull/344)
+* JIRA Ticket(s):
+    * [https://jira.cms.gov/browse/AB2D-1862](https://jira.cms.gov/browse/AB2D-1862)
+
+This proposal will suggest adding an OR clause to queries that gather the contract to patient mapping in order
+to improve efficiency on both the client and server. 
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+* [RFC Proposal](#rfc-proposal)
+* [Table of Contents](#table-of-contents)
+* [Motivation](#motivation)
+* [Proposed Solution](#proposed-solution)
+    * [Proposed Solution: Detailed Design](#proposed-solution-detailed-design)
+    * [Proposed Solution: Unresolved Questions](#proposed-solution-unresolved-questions)
+    * [Proposed Solution: Drawbacks](#proposed-solution-drawbacks)
+    * [Proposed Solution: Notable Alternatives](#proposed-solution-notable-alternatives)
+* [Future Possibilities](#future-possibilities)
+
+## Motivation
+[Motivation]: #motivation
+
+AB2D has long running export jobs where the amount of time the job takes is critical for customer happiness. In order to
+lessen the amount of time a job takes to run, we will need to be able to improve performance in every section of our code.
+One part that takes a long time is the call to the Patient URL that maps contracts to beneficiaries on a monthly basis.
+Being able to query multiple months in the same call will reduce the load on the BFD database since it will reduce the 
```

Comment:
We should include proving this assumption with performance tests during the implementation phase.

Reply:
Comparing this implementation to the master branch will be a good validation test I believe.